generator/C/include_v2.0/mavlink_types.h

```diff
@@ -15,6 +15,10 @@ namespace mavlink {
 // Macro to define packed structures
 #ifdef __GNUC__
   #define MAVPACKED( __Declaration__ ) __Declaration__ __attribute__((packed))
+#elif defined __arm__
```

Comment:
Could we use the macro __ARMCC_VERSION instead, which is the predefined macro for ARM KEIL compiler, while __UVISION_VERSION is related with MDK IDE environment.